visualizations/frontend/other/TTable.vue

```diff
@@ -5,6 +5,7 @@
     <!-- Title -->
     <div v-if="showTableHeader" class="tableHeaderContainer">
       <div class="title">
+        <div class="titlePrefix"><slot name="titlePrefix"></slot></div>
```

Comment:
```suggestion
        <div class="title__prefix"><slot name="titlePrefix"></slot></div>
```

It's not being used, but I recall Mehak saying that we should standardise around BEM. It's option, of course.

Reply:
Excellent point, thanks Sara!